db/migrate/20091218235406_seed_staged_content_for_affiliates.rb

```diff
@@ -1,6 +1,6 @@
 class SeedStagedContentForAffiliates < ActiveRecord::Migration
   def self.up
-    Affiliate.all.each { |aff| aff.update_attributes( :staged_domains => aff.domains, :staged_header => aff.header, :staged_footer => aff.footer) }
+    Affiliate.all.each { |aff| aff.update( :staged_domains => aff.domains, :staged_header => aff.header, :staged_footer => aff.footer) }
```

Comment:
Please undo all the changes to the files in `db/migrate`. We should avoid updating any of the historical migration files.